app/liquid_tags/forem_tag.rb

```diff
@@ -1,5 +1,5 @@
 module ForemTag
-  REGISTRY_REGEXP = %r{#{URL.url}/?}
+  REGISTRY_REGEXP = %r{#{URL.url.gsub("\.", "\\.")}/\b([\w-]+)?}
```

Comment:
I don't know what the full overlap is between valid DNS characters and regex special characters, so this line probably does cover what matters but I'm not sure. However, just in case, and also for readability (because regexes need all the help they can get in that department), this might help:

```suggestion
  REGISTRY_REGEXP = %r{#{Regexp.escape(URL.url)}/\b([\w\-]+)?}
```

Examples:

```
[1] pry(main)> Regexp.escape("https://dev.to")
=> "https://dev\\.to"
[2] pry(main)> Regexp.escape("https://my-cool-forem.forem.cloud")
=> "https://my\\-cool\\-forem\\.forem\\.cloud"
[3] pry(main)> URL.url
=> "http://jamie-mbp.local:3000"
[4] pry(main)> Regexp.escape URL.url
=> "http://jamie\\-mbp\\.local:3000"
```

I don't think the `-` will matter in a regex outside of a character-class matcher. But also, in your character-class matcher in that regex I saw something that doesn't seem to cause any issues now, but might cause some later if this were to change, so I escaped that `-` inside it. Specifically, the `-` inside a character-class matcher indicates a range, but I think it gets interpreted correctly right now because there's nothing on the righthand side of it.

Reply:
Interesting Jamie! I do think your approach is more readable.